.github/workflows/test.yml

```diff
@@ -235,6 +235,7 @@ jobs:
           - mysql
           - mariadb
           - mssql
+          - cockroachdb
```

Comment:
```suggestion
```

I don't think we have yet these integration-tests (in https://github.com/prisma/prisma/tree/main/packages/integration-tests) so we would need to remove that for now

Reply:
How do I get it to run https://github.com/prisma/prisma/blob/d267b1756cddd26b6ce4977601032b1331474443/packages/client/src/__tests__/integration/happy/blog-env-cockroachdb/test.ts#L71

Reply:
Naming might be confusing here, "integration-tests" is a different package and different tests, here `prisma/packages/client/src/__tests__/integration/happy/blog-env-cockroachdb/test.ts` is part of client package and with be matched by the structure/name automatically and will be added to the tests run without doing anything else.

Reply:
Ran here using binary engine and passed
https://github.com/prisma/prisma/runs/4580857834?check_suite_focus=true#step:9:812

Ran here using library engine and passed
https://github.com/prisma/prisma/runs/4580857643?check_suite_focus=true#step:9:829